Userland/Libraries/LibELF/DynamicLoader.cpp

```diff
@@ -210,11 +213,42 @@ void DynamicLoader::do_main_relocations()
         case RelocationResult::Success:
             break;
         }
+    });
+
+    auto fixup_trampoline_pointer = [&](DynamicObject::Relocation const& relocation) {
+        VERIFY(relocation.type() == R_X86_64_JUMP_SLOT || relocation.type() == R_AARCH64_JUMP_SLOT);
+        if (image().is_dynamic())
+            *((FlatPtr*)relocation.address().as_ptr()) += m_dynamic_object->base_address().get();
     };
 
-    do_relr_relocations();
-    m_dynamic_object->relocation_section().for_each_relocation(do_single_relocation);
-    m_dynamic_object->plt_relocation_section().for_each_relocation(do_single_relocation);
+    if (m_dynamic_object->must_bind_now()) {
```

Comment:
The comment about "passing in" BIND_NOW was lost. 

```
 // FIXME: Or BIND_NOW flag passed in?
```
Does it no longer apply?

Reply:
`git blame` says this comment was written by you... did you mean the `LD_BIND_NOW` environment variable?

Reply:
... assuming that's what you meant, I added the comment back.